diff --git a/lua/lazy/core/cache.lua b/lua/lazy/core/cache.lua
index c9313cd..159a092 100644
--- a/lua/lazy/core/cache.lua
+++ b/lua/lazy/core/cache.lua
@@ -214,7 +214,6 @@ function M.load(modpath, opts)
   chunk, err = Cache._loadfile(modpath, opts.mode, opts.env)
   if chunk then
     entry.chunk = string.dump(chunk)
-    Cache.write(modpath, entry)
   end
   M.track("load", start)
   return chunk, err
