diff --git a/lua/lazy/core/config.lua b/lua/lazy/core/config.lua
index d2b3146..8d39712 100644
--- a/lua/lazy/core/config.lua
+++ b/lua/lazy/core/config.lua
@@ -219,7 +219,6 @@ function M.setup(opts)
       pattern = "VeryLazy",
       once = true,
       callback = function()
-        require("lazy.core.cache").autosave()
         require("lazy.view.commands").setup()
         if M.options.change_detection.enabled then
           require("lazy.manage.reloader").enable()
